### PR TITLE
Naiserdoc

### PR DESCRIPTION
**Pull request by ninabakken**
Naiser naisdevice installation guide

### CODE REVIEW

**Comment by Starefossen** [comment]
@toby1knby u fix? ☝🏻

**Review by toby1knby — APPROVED** [review]
Lokks "Naiser" To Me

File: docs/device/install.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Installation
 
+!!!tip
+We have made a [nais'er installation guide](naiserinstall.md) if you think you may need a little extra help.
```
Comment (tronghn) [inline docs/device/install.md:4]:
```suggestion
!!! tip
    We have made a [nais'er installation guide](naiserinstall.md) if you think you may need a little extra help.
```

File: docs/device/naiserinstall.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
+# A nais'er installation guide
+
+!!!warn
```
Comment (tronghn) [inline docs/device/naiserinstall.md:3]:
```suggestion
!!! warn
```

File: docs/device/naiserinstall.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+
+## Installation steps, regardless of your device
+
+### Install Kolide agent
+
+The Kolide agent will be added to your Slack app, and let you know when there are recommended updates or security issues you need to address - and how to address them. The apps are found in the bottom left of your Slack app (scroll, scroll, scroll). 
+
+!!!warn
+The issues reported by Kolide must be addressed - these remediations have been vetted by the NAIS team and should be followed. Depending on the issue, you might lose naisdevice connectivity if an issue is left unresolved for a sufficient length of time. If you run into problems, you can always ask in the [Slack channel #naisdevice](https://nav-it.slack.com/archives/C013XV66XHB)
```
Comment (tronghn) [inline docs/device/naiserinstall.md:61]:
```suggestion
!!! warn
    The issues reported by Kolide must be addressed - these remediations have been vetted by the NAIS team and should be followed. Depending on the issue, you might lose naisdevice connectivity if an issue is left unresolved for a sufficient length of time. If you run into problems, you can always ask in the [Slack channel #naisdevice](https://nav-it.slack.com/archives/C013XV66XHB)
```

**Review by tronghn — APPROVED** [review]

**Pull request merged**